src/axolotl/utils/chat_templates.py

```diff
@@ -2,9 +2,15 @@
 This module provides functionality for selecting chat templates based on user choices.
 These templates are used for formatting messages in a conversation.
 """
+import logging
 
+LOG = logging.getLogger("axolotl.utils.chat_templates")
 
-def chat_templates(user_choice: str):
+_DEFAULT_TEMPLATE_CHOICE = "tokenizer_default"
+_DEFAULT_FALLBACK_CHATML_TEMPLATE_CHOICE_PREFIX = "tokenizer_default_fallback_"
+
+
+def chat_templates(user_choice: str, tokenizer=None):
```

Comment:
I had initially similarly included the tokenizer as an argument to chat template, but was trying to minimize the interdependency on this module with tokenizers. Seems like this is a reasonable workaround for now. thanks!